lib/wifi_user/use_case/contact_sanitiser.rb

```diff
@@ -30,6 +30,6 @@ def first_match(match)
   def internationalise_phone_number(phone_number)
     phone_number = '44' + phone_number[1..-1] if phone_number[0..1] == '07'
     phone_number = '+' + phone_number unless phone_number[0] == '+'
-    phone_number
+    phone_number.delete(' ')
```

Comment:
Given this isn't related to internationalisation, it might be best to break this out to a different function.

Perhaps in a way so the regex doesn't need changing?
```
internationalize(phone_match(strip(contact))) 
```